Gemfile.lock

```diff
@@ -73,16 +74,32 @@ GEM
     nokogiri (1.10.9)
       mini_portile2 (~> 2.4.0)
     parallel (1.19.1)
+    parser (2.7.1.3)
```

Comment:
This doesn't seem to be a dependency for `rubocop`, could you please elaborate?

Reply:
This was added when I did `bundle install`.   The [gemspec](https://github.com/rubocop-hq/rubocop/blob/master/rubocop.gemspec) for Rubocop requires it, and so it is not surprising to see it listed here as a dependency (and for Rubocop).  Specifically, line 37 states: `s.add_runtime_dependency('parser', '>= 2.7.0.1')`